wget/wget.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: wget
+spec:
+  params:
+   - name: url
+     description: The url we want to download file from
+     type: string
+     default: ""
+   - name: diroptions
+     description: The directory path we want to save file to
+     type: array
+     default: []
+   - name: options
+     description: The arguments to pass to wget
+     type: array
+     default: []
+   - name: filename
+     description: The filename we want to change our file to
+     type: string
+     default: ""
+   - name: wget-option
+     description: The wget docker image to be used
+     type: string
+     default: "buildpack-deps:stable-curl"
```

Comment:
Did you mean stable-wget, here instead?

Reply:
No, it is actually stable-curl, because this image does not have a wget flag version, its wget is included in this curl image. Dibyo already mentioned this in the comment above.

Reply:
I've also ran the test to make sure it works

Reply:
ah okay! thanks for checking!